lib/util.js

```diff
@@ -199,6 +199,13 @@ function deprecate(text) {
   /* eslint-enable */
 }
 
+function deprecateWrap(text, fn, that) {
```

Comment:
We don't care about `this` for these functions. So I guess we don't need to rebind the functions now, but we may need to in the future. I'll remove it until we need it.